data/en/fileclose.json

```diff
@@ -15,5 +15,13 @@
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/fileclose"}
 	},
 	"links": [],
-	"examples": []
+	"examples": [
+		{
+			"title":"Opens a file, reads a line then closes it.",
+			"description":"",
+			"code":"// Open File\nvar fileObject = fileOpen(\"/path/to/file.txt\");\n\n// Perform Actions\ntry\n{\n\t// Read Line\n\twriteOutput(fileReadLine(fileObject));\n}\n\n// Error Handling\ncatch(any ex)\n{\n\t// Report Exception\n\twriteDump(ex);\n}\n\n// Always Close\nfinally\n{\n\t// Close File\n\tfileClose(fileObject);\n}",
+			"result":"",
+			"runnable":true
```

Comment:
Let's set `"runnable": false` here since trycf does not let you use the file system functions.